src/components/FeatureViews/TeachingLog/TeachingLogContainer.js

```diff
@@ -264,22 +281,30 @@ export default function TeachingLogContainer() {
 								<TableCell className={classes.slHeadCell}>
 									Course Title
 								</TableCell>
-								<TableCell className={classes.slHeadCell}>
-									Total Hours
+								<TableCell className={classes.slHeadCellHours}>
+									Hours
 								</TableCell>
 							</TableRow>
 						</TableHead>
+					</Table >
+					<div className = {classes.slBody} >
+						<Table>					
+							<TableBody className = {classes.slBody}>
+								{
+									sessions.length > 0 && summaryLog.map(({title, hours, grade, courseId}) =>
+										<SummaryEntry key={courseId} title={title} hours={hours} grade={grade}/>)
+								}
+							</TableBody>
+						</Table>
+					</div>
+					<Table className = {classes.slFooter}>
 						<TableBody>
-							{
-								sessions.length > 0 && summaryLog.map(({title, hours, grade, courseId}) =>
-									<SummaryEntry key={courseId} title={title} hours={hours} grade={grade}/>)
-							}
 							<TableRow>
 								<TableCell>
 									<b>Total Hours</b>
 								</TableCell>
-								<TableCell>
-									{sessions.length > 0 && summaryLog.reduce((acc, course) => (acc.hours + course.hours))}
+								<TableCell className = {classes.TotalHoursNumber}>
+									<b>{sessions.length > 0 && summaryLog.reduce((acc, course) => (acc + course.hours), 0)}</b>
```

Comment:
avoid using the `<b>` tag. since you have a css class already, centralize all styling there

Reply:
This has been done, will be included when test is finished and pull request is updated.